src/components/Editor/components/MyWork/components/MyWorkEditorArea/index.js

```diff
@@ -105,13 +102,13 @@ export default class MyWorkEditorArea extends Component {
                                     data-intro="Describe your work by typing directly 
                                         or using the microphone to record an explanation of your work (required)."
                                     aria-label="Use the microphone button or type to explain your work (required)"
-                                    value={this.state.textAreaValue}
-                                    onChange={e => this.setState({ textAreaValue : e.target.value })}
+                                    value={this.props.textAreaValue}
+                                    onChange={value => this.props.textAreaChanged(value.target.value)}
                                     >
                                 </textarea>
                                 <SpeechToTextButton
-                                    textAreaValue={this.state.textAreaValue}
-                                    setTextAreaValue={value => this.setState({textAreaValue: value})} />
+                                    textAreaValue={this.props.textAreaValue}
+                                    setTextAreaValue={value => this.props.textAreaChanged(value)} />
```

Comment:
shouldn't it be different callback than in the editorArea? If any callback is needed here